README.md

```diff
@@ -24,6 +24,7 @@ description: High Performance Log and Metrics Processor
   * More than 80 built-in plugins available
   * Extensibility
     * Write any input, filter or output plugin in C language
+    * WASM: write [WASM Filter Plugins](development/wasm-filter-plugin.md) or [WASM Input Plugins](development/wasm-input-plugins.md)
```

Comment:
Should `write be here?
```suggestion
    * WASM: [WASM Filter Plugins](development/wasm-filter-plugin.md) or [WASM Input Plugins](development/wasm-input-plugins.md)
```

Reply:
@cosmo0920 please validate this.

Reply:
Ah, it needn't to use it here. I removed it.